packages/dapp/src/helpers/listingEvents.ts

```diff
@@ -24,9 +24,13 @@ export async function initializeSubscriptions(dispatch: Dispatch<any>): Promise<
 
 export async function getNewsroom(dispatch: Dispatch<any>, address: EthAddress): Promise<void> {
   const civil = getCivil();
+  const user = civil.userAccount;
   const newsroom = await civil.newsroomAtUntrusted(address);
   const newsroomWrapper = await newsroom.getNewsroomWrapper();
   dispatch(addNewsroom(newsroomWrapper));
+  if (user && newsroomWrapper.data.owners.includes(user)) {
```

Comment:
Ah cool. I should prob update the check that displays `Deposit/Withdraw/Exit Listing` on the Listing page for the Newsroom's owners to use this.